TombEngine/Specific/level.cpp

```diff
@@ -184,9 +184,9 @@ void LoadItems()
 			item->TriggerFlags = ReadInt16();
 			item->Flags = ReadInt16();
 			item->Name = ReadString();
-
-			g_GameScriptEntities->AddName(item->Name, i);
-			g_GameScriptEntities->TryAddColliding(i);
+			
+			g_GameScriptEntities->AddName(item->Name, static_cast<short>(i));
```

Comment:
Conventions prefer C-style casts unless `static_cast` is specifically required. Is it required here?

Reply:
Not really. I forgot to remove it after reading the guidelines yesterday.